pallets/invulnerables/src/lib.rs

```diff
@@ -213,4 +263,31 @@ pub mod pallet {
             Ok(())
         }
     }
+
+    use pallet_session::SessionManager;
+    use sp_staking::SessionIndex;
+
+    /// Play the role of the session manager.
+    impl<T: Config> SessionManager<T::AccountId> for Pallet<T> {
+        fn new_session(index: SessionIndex) -> Option<Vec<T::AccountId>> {
+            log::info!(
+                "assembling new invulnerable collators for new session {} at #{:?}",
```

Comment:
We will see this log on every session, right? It shouldn't say new if the invulnerables did not change, that may be confusing. And since pallet_collator_selection did not print any logs, maybe we should remove it or move it to debug level.

```suggestion
                "Assembling invulnerable collators for new session {} at #{:?}",
```

Reply:
This is the exact behavior in the `collator-selection` pallet. Even if there is no new collators, it signals the new session.

Reply:
You are right that this log is copied from `collator-selection`, but when I run tanssi-node from master I don't see that log. Any ideas why? Maybe because it's from another crate? Or we didn't update invulnerables on every new session on master?

Reply:
checking...

Reply:
It is there:
```
2023-08-31 08:18:49 assembling new collators for new session 0 at #0
2023-08-31 08:18:49 assembling new collators for new session 1 at #0
```

Reply:
May be you looked for the exact string `assembling new invulnerable collators for new session`?

Reply:
Thank you, I see it now, indeed I was looking for "new invulnerable". We can keep this log as is then.